openspec/changes/fix-failure-alerting/tasks.md

```
@@ -0,0 +1,31 @@
+## 1. Reconciler: alert on all failure paths
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)